### ISSUE TITLE
Railguns produce excessicve amounts of noise.

### ISSUE THREAD

**Issue by Mecares** [CleverRaven/Cataclysm-DDA#20898]
Weapon | Ammo | Sound
----------|--------|-------
H&K G80 | H&K 12mm | 23
Coilgun | Flechette | 41
Nail Rifle | Flechette | 52 
Ferromagnetic rail rifle | Rebar rail | >210
Heavy rail rifle | Rebar rail| >250

Considering that they do not have a propellant but instead accelerate the projectile via magnetism the produced sound of the Ferromagnetic rail rifle and the Heavy rail rifle do look far too high.

**Comment by Rivet-the-Zombie** [comment]
They fire their projectiles at such a high speed that they incandesce as they travel through the air, so I'd guess that these guns sound volume has something to do with that.

**Comment by DangerNoodle** [comment]
How loud are these compared to non-fictional firearms, though?

**Comment by Solusphere** [comment]
Not to mention that the rounds break the sound barrier several times over.

**Comment by Mecares** [comment]
> How loud are these compared to non-fictional firearms, though?

Some Values to compare:

Weapon | Ammo | Sound
----------|--------|-------
Glock 19 | 9X19mm JHP | 79
Mossberg 500 | 00 Shots | 159
AR 15 | .223 Remingtons | 163
M60 | 7.62x51mm M80s | 212
Barrett M107A1 | .50 BMG | 237

**Comment by Regularitee** [comment]
Okay, I've roughly crunched some numbers for the railgun, and they don't make sense, so trying to realistically guess the sound is actually impossible. 

Simply put, a projectile of that mass (iron/steel, of 5-7kg) launched at sufficient speed to become incandescent from air friction in fraction of a second before it reaches a target would mean the recoil is equivalent to... well, something you'd compare ridiculous amounts of recoil to. A fully loaded freight train at full steam? A battleship ramming you? A small town falling on you? You get the picture.

On the other hand, one could claim that it glows red from the amount of energy spent by the weapon. Real-life railguns haven't become practical because the sheer voltages required to launch the projectile will melt and vaporize the rails themselves. Assuming some of that heat transfers to the projectile, it would already launch red hot. As such its speeds may not necessarily be extreme the description suggests or people imagine.

**Comment by DangerNoodle** [comment]
How much more range and armor pierce does the heavy railgun have relative to other firearms? It would be reasonable to guess how fast a real-life projectile should be going to get the results displayed by the in-game weapon, and model the sound after how loud a weapon of that nature is speculated to be.

I do know that homemade coilguns exist, with much lower velocities due to design limitations. But I am unsure how loud they are. One fast enough to break the sound barrier might not be all that loud, as even most standard .22 LR rounds are supersonic.

**Comment by Tharn** [comment]
The numbers don't make sense because 5-7kg is by my estimation a ridiculously oversized projectile for an anti-personnell railgun. Projectiles used in Navy trials are 10-15 kg and those are meant to drill right through armoured boats.

**Comment by CoroNaut** [comment]
Do we even have a balanced system of sound intensity similar to decibel intensity? Do these numbers match any form of equation to determine their actual values or are these numbers just constants?

If you are running a jackhammer, it can produce around 120dB and wearing good ear protection can lower that to 85dB. This reduces sound intensity by a factor of over 3000.

If there is a logarithmic scale similar to the decibel intensity with which we could apply real numbers to these weapons, I'm sure it would be more balanced. Having them as constants without any formula is just guessing at their actual sound intensity.

**Comment by Zireael07** [comment]
@CoroNaut: No, we don't and using decibels has been suggested once and Kevin said it's an end goal.

**Comment by CoroNaut** [comment]
Alright, cool. In any case, the sound intensity will have to be reworked for the 1 second turn idea that kevingranade wants to implement.

**Comment by Solusphere** [comment]
> The numbers don't make sense because 5-7kg is by my estimation a ridiculously oversized projectile for an anti-personnell railgun.

For the record, a single steel rail weighs 520 grams, which is much more reasonable. You were probably looking at a stack of them, and mistaking the total weight for individual weight.

**Comment by Leland** [comment]
See #14197 for a talk on moving to decibels.

**Comment by kevingranade** [comment]
Feel free to set the sound level somewhere in the range of 50-75, anywhere in there seems reasonable pending a db based overhaul.

**Comment by ZhilkinSerg** [comment]
All nail/rail guns are now produce about 20 sound in my tests. Should it be fixed or the issue can be closed?
